### PR TITLE
Quickfix errors with roles check in the back-end

### PR DESCRIPTION
**Pull request by goreck888**
This commit contains check if user exist by checking the role
of user. It's necessary because ruby tries to call not existing
method fo a NilClass.

### CODE REVIEW

File: app/policies/backoffice/category_policy.rb [hunk 1]
```diff
@@ -40,6 +40,6 @@ def permitted_attributes
   private
 
     def service_portfolio_manager?
-      user.service_portfolio_manager?
+      user && user.service_portfolio_manager?
```
Comment (martaswiatkowska) [inline app/policies/backoffice/category_policy.rb]: user&.service_portfolio_manager? isn't simpler ?

Comment (goreck888) [reply app/policies/backoffice/category_policy.rb]: Yes, it is. I didn't know that notation. Thanks!

**Comment by agpul** [comment]
Can you help me and tell me how can I test this PR?

**Comment by goreck888** [comment]
Try to go to the backoffice without login and ask @wziajka about logs.

**Pull request merged**